addons/mail/static/tests/tours/discuss_channel_as_guest_tour.js

```diff
@@ -17,6 +17,10 @@ registry.category("web_tour.tours").add("discuss_channel_as_guest_tour.js", {
             trigger: "button[title='Join Channel']",
             run: "click",
         },
+        {
+            content: "Check that we are on not in a call",
+            trigger: "button[title='Start Call']",
+        },
```

Comment:
Runbot red because "Luigi" internal user is in the call, and button label changes from "Start Call" to "Join Call".

Change of label should be enough for runbot to be happy.